resharper/debugger/debugger/Values/Renderer/ChildrenRenderers/NodeObjectChildrenRenderer.cs

```diff
@@ -82,10 +81,23 @@ private IValueEntity[] GetChildrenImpl(IObjectValueRole<TValue> valueRole, IValu
                 return false;
             }
 
-            returnedPropertyRole = new SimpleValueReference<TValue>(
-                    role.CallInstanceMethod(method),
-                    role.ValueReference.OriginatingFrame, ValueServices.RoleFactory)
-                .AsObjectSafe(options);
+            if (method.Parameters.Any())
+            {
+                var frame = role.ValueReference.OriginatingFrame;
+                var param = ValueServices.ValueFactory.CreatePrimitive(frame, options, false); // todo: how to read param default value?
```

Comment:
If we are sure that method parameters are primitive, we can get their names as ```parameter.Type.FullName``` and pass their default values
.